StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetConfiguration.swift

```diff
@@ -61,8 +61,7 @@ extension PaymentSheet {
         var isSelectedByDefault: Bool {
             switch self {
             case .automatic:
-                // only enable the save checkbox by default for US
-                return Locale.current.stp_regionCode == "US"
+                return false
```

Comment:
We should probably add this to our list of "things to break" in the next major version

Reply:
I like having .automatic around just in case, maybe someday we can control it server-side